src/jukebox/components/synchronisation/sync_shared/__init__.py

```diff
@@ -0,0 +1,355 @@
+"""
```

Comment:
This is quite a long file. You could outsource helper functions into a separate file to optimize readability (see `components/rfid`)

Reply:
I will look into this